src/model/objection.company.profile.ts

```diff
@@ -0,0 +1,13 @@
+export interface ObjectionCompanyProfile {
+  companyName: string;
+  companyNumber: string;
+  companyStatus: string;
+  companyType: string;
+  address: {
+    line_1: string;
+    line_2: string;
+    postCode: string;
+  };
+  jurisdiction: string;
```

Comment:
I can't see where jurisdiction is used (at least at present). Shouldn't we include this later when we know it's needed?

Reply:
ok